Makefile

```diff
@@ -80,17 +82,17 @@ S_FILES  := $(foreach dir,$(SRC_DIRS),$(wildcard $(dir)/*.s))
 
 # Versions J and below used the C matrix math implementations
 MGU_MATRIX_FILES := mtxcatf normalize scale translate
-ifneq ($(filter $(VERSION),D E F G H I J),)
-S_FILES := $(filter-out $(addprefix src/mgu/,$(MGU_MATRIX_FILES:=.s)),$(S_FILES))
-else
+ifeq ($(MGU_ASM), 1)
 C_FILES := $(filter-out $(addprefix src/gu/,$(MGU_MATRIX_FILES:=.c)),$(C_FILES))
+else
+S_FILES := $(filter-out $(addprefix src/mgu/,$(MGU_MATRIX_FILES:=.s)),$(S_FILES))
 endif
 
 C_O_FILES := $(foreach f,$(C_FILES:.c=.o),$(BUILD_DIR)/$f)
 S_O_FILES := $(foreach f,$(S_FILES:.s=.o),$(BUILD_DIR)/$f)
 O_FILES   := $(S_O_FILES) $(C_O_FILES)
 
-AR_OBJECTS := $(shell cat base/$(VERSION)/$(TARGET).txt)
+AR_OBJECTS := $(shell cat base/$(TARGET).txt)
```

Comment:
This make variable isn't actually used anymore so it can be removed. This also allows removing the entire base directory